cmd/qemu-ifdown/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+var DefaultConfPath string
+
+// todo: error handling
```

Comment:
At least all the errors from ``cmd.Run()`` must be caught. Running external program has many error cases so that it will make debugging integration test harder.